packages/@o3r/test-helpers/src/utilities/package-manager.ts

```diff
@@ -297,7 +297,7 @@ export function setPackagerManagerConfig(options: PackageManagerConfig, execAppO
         execFileSync('yarn', ['config', 'set', 'enableGlobalCache', 'true'], execOptions);
         execFileSync('yarn', ['config', 'set', 'globalFolder', options.globalFolderPath], execOptions);
       }
-      execFileSync('yarn', ['config', 'set', 'nodeLinker', 'pnp'], execOptions);
+      execFileSync('yarn', ['config', 'set', 'nodeLinker', 'node-modules'], execOptions);
```

Comment:
```suggestion
     // TODO reactivate pnp once https://github.com/AmadeusITGroup/otter/issues/2620 is resolved
      execFileSync('yarn', ['config', 'set', 'nodeLinker', 'node-modules'], execOptions);
```
Issue for pnp has been created: https://github.com/AmadeusITGroup/otter/issues/2620